megamek/src/megamek/client/ui/swing/BoardEditor.java

```diff
@@ -1751,6 +1761,17 @@ private void showSettings() {
         // Show the settings dialog.
         setdlg.setVisible(true);
     }
+    
+    private void freshBoardSettings() {
```

Comment:
I'm not sure about this name, as it took me a second to realise what it meant.

Reply:
Do you have a suggestion? Its not necessarily a new board, not necessarily a loaded board. updateUiForFreshBoard...

Reply:
refreshBoardSettings? Not super fitting, but better. refreshBoardUI?